flow-typed/npm/danger_v0.x.x.js

```diff
@@ -0,0 +1,193 @@
+
```

Comment:
It will! Definitely, I want to get shipping PRs to Flow-Typed (+DefinitelyTyped) automated from my end as a part of deploying DangerJS